modules/custom/activity_basics/src/Plugin/ActivityAction/PublishActivityAction.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Drupal\activity_basics\Plugin\ActivityAction;
+
+use Drupal\activity_creator\Plugin\ActivityActionBase;
+
+/**
+ * Provides a 'PublishActivityAction' activity action.
+ *
+ * @ActivityAction(
+ *  id = "publish_entity_action",
+ *  label = @Translation("Action that is triggered when a entity is published"),
```

Comment:
Should be three spaces between asterisk and field name.

Reply:
Fixed